keyboards/gergo/keymaps/numist/keymap.c

```diff
@@ -0,0 +1,52 @@
+/* Good on you for modifying your layout! if you don't have
+ * time to read the QMK docs, a list of keycodes can be found at
+ *
+ * https://github.com/qmk/qmk_firmware/blob/master/docs/keycodes.md
+ *
+ * There's also a template for adding new layers at the bottom of this file!
+ */
+
+#include QMK_KEYBOARD_H
+
+#define BASE 0  // default layer
+#define FN 1    // macros/motion
+#define NUMB 2  // numbers
+
+// Blank template at the bottom
+
+#define DYNAMIC_MACRO_SIZE 256
+#define DYNAMIC_MACRO_RANGE SAFE_RANGE
+#include "dynamic_macro.h"
+
+#define M_RECD1 DYN_REC_START1
+#define M_STOP1 DYN_REC_STOP
+#define M_PLAY1 DYN_MACRO_PLAY1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+// Keymap 0: Base layer
+[BASE] = LAYOUT_gergo(
+KC_GRAVE,              KC_Q,  KC_W,   KC_F,   KC_P, KC_B,                  /**/                   KC_J, KC_L, KC_U,    KC_Y,   KC_SCLN, KC_EQL,
+LT(NUMB, KC_PGUP),     KC_A,  KC_R,   KC_S,   KC_T, KC_G, KC_TAB,          /**/          KC_ENT,  KC_M, KC_N, KC_E,    KC_I,   KC_O,    KC_QUOT,
+MT(MOD_LSFT, KC_PGDN), KC_Z,  KC_X,   KC_C,   KC_D, KC_V, KC_ESC, KC_LBRC, /**/ KC_RBRC, KC_BSLS, KC_K, KC_H, KC_COMM, KC_DOT, KC_SLSH, KC_MINS,
+
+                                     KC_LCTL,  KC_LALT,  KC_LCMD, KC_LSFT, /**/ KC_BSPC, KC_RSFT,  LT(FN, KC_SPC),  MT(MOD_RCTL, KC_DEL)
+// TODO: MT taps KC_CAPS too quickly for the key hysteresis imposed by macOS; need to find a different solution for MT(MOD_RSFT, KC_CAPS).
```

Comment:
For this particular issue, you can add `#define TAP_HOLD_CAPS_DELAY <ms>` to your config.h.

Reply:
Follow-up question, if a keymap contains a `config.h` does that file replace the keyboard-defined `config.h` or just get included afterward?

IoW does a keymap's `config.h` have to start from a copy of its keyboard's `config.h`?

Reply:
Both are included, if you want to override a `#define` from the keyboard level, you first have to `#undef` it.